build/PackageInfo.xml

```diff
@@ -0,0 +1,6 @@
+<pkg-info format-version="2" identifier="io.lbry.LBRY.pkg" version="0.10.0" install-location="/" auth="root">
```

Comment:
Defines if the user needs to authenticate as an admin user to install the package